alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/endpoint/oauth/OAuthEndpointResponse.java

```diff
@@ -27,23 +27,16 @@
 import com.synopsys.integration.util.Stringable;
 
 public class OAuthEndpointResponse extends Stringable implements Serializable {
```

Comment:
We have a class called `AlertSerializableModel` that `extends Stringable` and `implements Serializable` so that we don't have to extend both like this.